e2e/benchmarks/local-benchmark/dump.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+function compareData(data1, data2, level = 0) {
+  if (level == 0) {
+    let notMatch = false;
+    try {
+      expectObjectsClose(data1, data2);
+    } catch (e) {
+      notMatch = true;
+    }
+    return notMatch;
+  } else if (level == 1) {
+    return JSON.stringify(data1) !== JSON.stringify(data2);
+  } else if (level == 2) {
+    return true;
+  }
+}
+
+function getGraphModel(model, benchmark) {
+  let graphModel = null;
+  if (benchmark === 'bodypix' || benchmark === 'posenet') {
+    graphModel = model.baseModel.model;
+  } else if (
+      benchmark === 'USE - batchsize 30' || benchmark === 'USE - batchsize 1') {
+    graphModel = model.model;
+  } else {
+    graphModel = model;
+  }
+
+  if (graphModel instanceof tf.GraphModel) {
+    return graphModel;
+  }
+  console.warn(`Model ${benchmark} doesn't support dump op!`);
+  return null;
+}
+
+async function getIntermediateTensorInfo(tensorsMap) {
+  if (!tensorsMap) {
+    return;
+  }
+  const jsonObject = {};
+  const keysOfTensors = Object.keys(tensorsMap);
+  for (let i = 0; i < keysOfTensors.length; i++) {
+    const key = keysOfTensors[i];
+    const dataArray = [];
+    jsonObject[key] = [];
+    for (let j = 0; j < tensorsMap[key].length; j++) {
+      const data = await (tensorsMap[key][j]).data();
+      dataArray.push(data);
+      jsonObject[key].push({
+        value: dataArray,
+        shape: tensorsMap[key][j].shape,
+        dtype: tensorsMap[key][j].dtype
+      });
+    }
+  }
+  return jsonObject;
+}
+
+async function saveObjectsToFile(
+    jsonObjects, backends, level, dumpCount, prefix) {
+  let newPrefix = '';
+  if (prefix !== '') {
+    newPrefix = `${prefix.replace(/\//g, '-')}_`;
+  }
+  if (((level < 2) && dumpCount) || (level === 2)) {
+    for (let i = 0; i < jsonObjects.length; i++) {
+      const object = jsonObjects[i];
+      const fileName = `${newPrefix}${backends[i]}.json`;
+      const a = document.createElement('a');
+      const file =
+          new Blob([JSON.stringify(object)], {type: 'application/json'});
+      a.href = URL.createObjectURL(file);
+      a.download = fileName;
+      a.click();
+      await sleep(150);
```

Comment:
I remember that we want to dump to one file instead of many files? So that we don't need to `await sleep(150)`?

Reply:
This is tried to make it works better with our automation tools. However, I still can not get stable result from this. So I will remove this.